mintpy/objects/gps.py

```diff
@@ -168,37 +172,43 @@ def get_gps_los_obs(insar_file, site_names, start_date, end_date,
             # calculate gps data value
             obj = GPS(site_name)
 
-            if obs_type == 'velocity':
+            if obs_type == 'velocity' or obs_type == 'SenD':
                 vel, dis_ts = obj.get_gps_los_velocity(geom_obj,
                                                        start_date=start_date,
                                                        end_date=end_date,
-                                                       gps_comp=gps_comp)
+                                                       ref_site=ref_site,
```

Comment:
I did not add referencing on purpose before, so that data in the CSV file could be more generic. Note that the referencing is applied as below right after this.

https://github.com/insarlab/MintPy/blob/6c7ad9ec07462022f021886b92f02164bf842ef6/mintpy/utils/plot.py#L1140

Reply:
Do you think perhaps it may be a better for the CSV to reflect the plot, especially since it may make it more convenient for the user to take the CSV and use it for their work through other programs/interfaces (e.g. RAiDER, QGIS, etc.).
I understand your rationale to keep it generic and it should be simple for the user to manually tweak the CSV to reflect the specified reference station, but then again it makes it much more convenient to take these outputs directly as part of a broader, automated routine or so. What do you think?

Reply:
Good point. Let keep `ref_site` then. 

One detail: I would recommend, inside `get_gps_los_obs()`, using a simple referencing as we have in `plot.plot_gps()`, instead of passing ref_site to `read_gps_los_velocity/displacement()` for faster speed and more robust, because `read_gps_los_displacement()` is grabbing the common acquisitions between GPS pairs. It's fine to compare time-series, which is my original purpose, but can be a potential issue for displacement/velocity when the two sites has sparse and uncommon dates.

Reply:
Ok implemented as suggested, I hope it's cleaner/simpler! However, not that I left alone the sections of the `read_gps_los_velocity/displacement()` functions which handle the ref station. I don't specify the reference stations in the respective function call, but I just left those functions alone.

Reply:
that sounds great.

Reply:
Now I remembered that: no-referencing is required to support spatially absolute comparison bwt. SAR and GPS. I have re-moved it to `plot.plot_gps()` again and add some comments for future reference.